src/components/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/search_results.css";
+
+const SearchResults = ({ results }) => {
+  if (!results?.length) {
```

Comment:
Nice inclusion of optional chaining! It isnt actually needed in this case as it happens, because you are always passing down an array (it may be an empty one, but its an array nonetheless). This means the length property will always return a value so it isnt needed, its more for use when trying to access a property of an object that may not exist and you want to mitigate against potential errors. Always worth a read on [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) if you want to know more though :)